docs/data/data-grid/master-detail/FullWidthDetailPanel.js

```diff
@@ -211,6 +211,11 @@ export default function FullWidthDetailPanel() {
         pinnedColumns={{ left: [GRID_DETAIL_PANEL_TOGGLE_FIELD] }}
         getDetailPanelHeight={getDetailPanelHeight}
         getDetailPanelContent={getDetailPanelContent}
+        sx={{
+          '& .MuiDataGrid-detailPanel': {
+            overflow: 'visible',
```

Comment:
I wonder if making `overflow: visible` a default style would have any caveats 🤔 
For instance, I tried it on the examples in the demo page and they seem to work fine.